src/west/manifest-schema.yml

```diff
@@ -118,3 +122,8 @@ mapping:
       west-commands:
         required: false
         type: str
+      # Additional manifest data to import from the manifest
+      # repository.
+      import:
```

Comment:
@tejlmand and I were discussing about this. I think you added the `import` key in the `self` section in order to be able to import additional `.yml` manifests that are present directly in the file system. This is useful for users adding local projects without having to modify the manifest file and also for CI in order to add ci-specific manifests.

I think there are several solutions to this problem:

- Have a hardcoded path `west_topdir/.manifests` that will always be loaded by default, without requiring explicitly setting it into the manifest itself
- The solution you propose here, but I would probably rename `import` in the `self` section to `local-import` or similar
- A new command-line options `--extra-manifests` that allows users to specify the same but via the command line (ideal for CI)

How do you see this proposal you have being used in the context of a downstream? If it's for CI, then the users will be exposed to a folder listed there that has no relevance to them and in the case of CI itself running, there will no indication of whether the extra CI repos have been cloned or not.

Reply:
>  I think you added the import key in the self section in order to be able to import additional .yml manifests that are present directly in the file system. 

It was actually added to enable splitting up the manifest content into multiple files within a single manifest repository:

https://builds.zephyrproject.org/zephyrproject-rtos/zephyr/20433/docs/guides/west/manifest.html#west-manifest-ex2-2

I'm not trying to address additional manifest data in a general place on the file system, i.e. outside of a repository known to west (either the manifest repository or a project).

Reply:
@mbolivar can you provide a brief update on this.
How is it used in current PR ?

Reply:
> @mbolivar can you provide a brief update on this.
> How is it used in current PR ?
> see the reason in this commit [7b204e9](https://github.com/zephyrproject-rtos/west/commit/7b204e900ce81b2715ca71a4fa12388ea5d0ae35)

Just in case it's not clear, the reason is for `self: import:`, i.e additional manifest files in the manifest repository itself, as shown in the example I linked to in the built documentation. That link has now expired, but I'll refresh it.